io/net/virt-net/lpm.py

```diff
@@ -66,6 +66,7 @@ def setUp(self):
         '''
         self.install_packages()
         self.rsct_service_start()
+        self.rmc_service_start()
```

Comment:
@abdhaleegit My thoughts: This is a workaround, so we should not be PASSing the test in this scenario.
We can check if the rsct services are active, and if inactive, set a flag. Restart only if necessary
At the end, we can check if flag is set, WARN/FAIL the test based on that. This also makes sure that LPM continues and failure is logged.

Let me know if this suggestion is fine with you.

Reply:
RMC service going down after a migration is a bug and I have raised it against HMC,   

for the test there will be a check for rmc after every migration and WARN test if rmc is inactive and also make sure the rmc is restarted for other test to continue (this makes sure the migration is complete even after warn)

@narasimhan-v

Reply:
With the latest code, I have taken care of above suggestion